integration/ldt_desired_properties_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+//go:build integration
+
+package integration
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/eclipse/ditto-clients-golang/model"
+	"github.com/eclipse/ditto-clients-golang/protocol/things"
+)
+
+type ldtDesiredPropertiesSuite struct {
+	localDigitalTwinsSuite
+
+	messagesFilter string
+	expectedPath   string
+}
+
+func (suite *ldtDesiredPropertiesSuite) SetupSuite() {
+	suite.SetupLdtSuite()
+	suite.messagesFilter = fmt.Sprintf("like(resource:path,'/features/%s/desiredProperties')", featureID)
+	suite.expectedPath = fmt.Sprintf("/features/%s/desiredProperties", featureID)
+}
+
+func (suite *ldtDesiredPropertiesSuite) TearDownSuite() {
+	suite.TearDownLdtSuite()
+	suite.TearDown()
+}
+
+func TestDesiredPropertiesSuite(t *testing.T) {
+	suite.Run(t, new(ldtDesiredPropertiesSuite))
+}
+
+func (suite *ldtDesiredPropertiesSuite) TestEventModifyOrCreateDesiredProperties() {
+	properties := map[string]interface{}{desiredProperty: value}
+	tests := map[string]struct {
+		command        *things.Command
+		expectedTopic  string
+		beforeFunction func()
+	}{
```

Comment:
This struct is repeated multiple times throughout the test cases. Could you move it to 'ldt_suite.go' and reuse it instead? Also, please adapt it as I commented below.